packages/core/src/odata/common/request-builder/batch/batch-change-set.ts

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+// TODO: Add specific request builder types as generics
+/**
+ * Representation of a batch change set, which holds a collection of write operations.
+ */
+export interface BatchChangeSet<T = any> {
```

Comment:
This generic type argument is not really used right? Is this there for compatibility to previous versions?

Reply:
No it is used, but it was too generic. Made it more specific, but this also needs to be updated once we have base types for all the request builders.